test/integration/scheduler_perf/config/performance-config.yaml

```diff
@@ -30,11 +30,17 @@
       initPods: 500
       measurePods: 1000
   - name: 5000Nodes
-    labels: [performance, fast]
```

Comment:
in which jobs will this test case run, after the change?

Reply:
In no test, but could be run additionally by setting the test selector to `fast`